deployment/entity-service/values.yaml

```diff
@@ -118,36 +118,38 @@ api:
     enabled: false
 
     ## Ingress annotations
-    annotations: # {}
-     kubernetes.io/ingress.class: nginx
+    annotations: {}
+      ## Suggested annotations
+      ## To handle large uploads we increase the proxy buffer size
+      #ingress.kubernetes.io/proxy-body-size: 4096m
 
-     ## To handle large uploads we increase the proxy buffer size
-     ingress.kubernetes.io/proxy-body-size: 4096m
-     nginx.ingress.kubernetes.io/proxy-body-size: 4096m
+      ## Redirect to ssl
+      #ingress.kubernetes.io/force-ssl-redirect: "true"
 
-     nginx.ingress.kubernetes.io/proxy-connect-timeout: "60"
-     nginx.ingress.kubernetes.io/proxy-send-timeout: "60"
-     nginx.ingress.kubernetes.io/proxy-read-timeout: "60"
-     ## Example ingress annotations for certmanager
-     certmanager.k8s.io/cluster-issuer: letsencrypt-cluster-issuer
-     ## Redirect to ssl
-     ingress.kubernetes.io/force-ssl-redirect: "true"
+      ## Deprecated but common
+      ## https://kubernetes.io/docs/concepts/services-networking/ingress/#deprecated-annotation
+      # kubernetes.io/ingress.class: ""
+
+    path: /
+    pathType: Prefix
 
     ## Entity Service API Ingress hostnames
     ## Must be provided if Ingress is enabled
-    hosts:
-     - beta.anonlink.data61.xyz
```

Comment:
@wilko77 I suggest these removed/commented out values might go into a private data61 deployment repo

Reply:
Wasn't there a problem with uploads and timeouts? Are those nginx timeouts essential for the ES to work? If so, we should ensure that they are either set to sensible defaults, or there are big hints in the doc that these values need to be set.

Reply:
Ok I'll document the timeout issues we ran into and the workaround we used for nginx ingress controllers.

The issue is that nginx is just one of many options for a Kubernetes cluster's "ingress controller".  These settings were correct for deployments to the Data61 development/demo cluster but the chart's values.yaml are defaults for everyone.